py/release.py

```diff
@@ -165,13 +165,19 @@ def build_operator_image(root_dir,
   # Building dashboard's front-end
   util.run(["yarn", "--cwd", "{}/dashboard/frontend".format(root_dir), "build"])
 
+  # If the release is not done from a Linux machine
+  # we need to grab the artefacts from /bin/linux_amd64
+  bin_path = "bin"
+  if platform.system() != "Linux":
+    bin_path += "/linux_amd64"
+
```

Comment:
If I am doing `python -m py.release local` from a non linux machine, the build output will be located in `$GOPATH/bin/linux_amd64`, not in `$GOPATH/bin`